.golangci.yml

```diff
@@ -1,63 +1,168 @@
-    # switch
-    default-signifies-exhaustive: true
+    shadow: true
+    enable-all: true
+    disable:
+      - fieldalignment
   revive:
-    rules:
-      - name: var-naming
-        disabled: true
-  whitespace:
-    # Enforces newlines (or comments) after every multi-line if statement.
-    # Default: false
-    multi-if: true
-
-    # Enforces newlines (or comments) after every multi-line function signature.
-    # Default: false
-    multi-func: true
-
+    confidence: 0
+  prealloc:
+    simple: false
+    range-loops: true
+    for-loops: true
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - whyNoLint
+      - unnamedResult
+  errorlint:
+    errorf: true
+  nolintlint:
+    require-explanation: true
+    require-specific: true
+  tagalign:
+    align: false
+run:
+  timeout: 15m
+service:
+  golangci-lint-version: "1.57"
 linters:
+  disable-all: true
   enable:
+    - asasalint
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - containedctx
+    - contextcheck
+    - copyloopvar
+    - cyclop
+    - decorder
+    - depguard
+    - dogsled
+    - dupl
+    - dupword
+    - durationcheck
+    - errcheck
+    - errchkjson
+    - errname
+    - errorlint
+    - execinquery
     - exhaustive
     - exportloopref
+    - forbidigo
+    - forcetypeassert
     - gci
+    - ginkgolinter
+    - gocheckcompilerdirectives
+    - gochecknoglobals
+    - gochecknoinits
+    - gochecksumtype
+    - gocognit
+    - goconst
+    - gocritic
+    - gocyclo
+    - godox
+    - goerr113
     - gofmt
+    - gofumpt
+    - goheader
+    - goimports
+    - gomnd
+    - gomoddirectives
+    - gomodguard
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - gosmopolitan
+    - govet
+    - grouper
+    - importas
+    - ineffassign
+    - interfacebloat
+    - intrange
+    - ireturn
+    - lll
+    - loggercheck
+    - maintidx
+    - makezero
+    - mirror
     - misspell
+    - nakedret
+    - nestif
     - nilerr
+    - nilnil
     - nlreturn
+    - noctx
+    - nolintlint
+    - nonamedreturns
+    - nosprintfhostport
+    - paralleltest
+    - perfsprint
+    - prealloc
+    - predeclared
+    - promlinter
+    - protogetter
+    - reassign
     - revive
+    - rowserrcheck
+    - sloglint
+    - spancheck
+    - sqlclosecheck
+    - staticcheck
+    - stylecheck
+    - tagalign
+    - tenv
+    - testableexamples
+    - testifylint
+    - thelper
+    - tparallel
     - unconvert
     - unparam
+    - unused
+    - usestdlibvars
+    - varnamelen
     - wastedassign
-    - execinquery
-    - nonamedreturns
-    - nosprintfhostport
-    - containedctx
-    - errchkjson
-    - contextcheck
-    - errname
-    - wastedassign
-    - nilerr
-    - gosec    
-    - govet
     - whitespace
-
+    - wrapcheck
+    - wsl
+    - zerologlint
 issues:
-   exclude-rules:
-    - path: _test\.go
+  exclude-rules:
+    - path: (.+)_test\.go
```

Comment:
Is it possible to remove some linter from the example?

Something like:

```
 - path: cmd/dfns_example/(.+).go
      linters:
        - funlen
        - goconst
        - interfacer
        - dupl
        - lll
        - goerr113
        - varnamelen
        - errcheck
        - gocritic
        - cyclop
        - wrapcheck
        - gocognit
        - contextcheck
        - nlreturn
        - gochecknoglobals
        - errorlint
```

(at least err113, as it makes no sense for an example to make error static)